tests/console/hostname.pm

```diff
@@ -26,10 +26,14 @@ use strict;
 use warnings;
 use testapi;
 use utils;
+use version_utils "is_sle";
 
 sub run {
     select_console 'root-console';
 
+    # Since 15SP1 DHCLIENT_SET_HOSTNAME must be disable bsc#1153402
+    assert_script_run "sed -i -e 's|^DHCLIENT_SET_HOSTNAME=.*|DHCLIENT_SET_HOSTNAME=\"no\"|' /etc/sysconfig/network/dhcp" if (is_sle('>15'));
```

Comment:
Shouldn't be for `15-SP1+`? Or is recent Maintenance Updates also changed this behavior?

Reply:
`is_sle('>15')` is the same as `is_sle('15-SP1+')` or I'm missing something?
Anyway this fix is for 15SP1 and 15SP2.

Reply:
You're right... this is why I prefer the `15-SP1+` way :D as I can directly read the minimal version, but that's a matter of taste :-)